ooniprobe.xcodeproj/project.pbxproj

```diff
@@ -744,12 +762,12 @@
 					/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include,
 				);
 				INFOPLIST_FILE = ooniprobe/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited)";
 				LIBRARY_SEARCH_PATHS = "$(inherited)";
 				PRODUCT_BUNDLE_IDENTIFIER = org.openobservatory.ooniprobe;
 				PRODUCT_NAME = ooniprobe;
-				PROVISIONING_PROFILE = "9c5b2b6c-065a-4427-96fc-1cba88a959e8";
+				PROVISIONING_PROFILE = "de5a74e0-ac7a-48a3-b51c-fa0b65ad0759";
```

Comment:
@lorenzoPrimi do you know why this changed?

Reply:
new provisioning sent to me by nick